src/dnnl/dnnl_graph_compiler.cpp

```diff
@@ -1,9 +1,21 @@
 #include "dnnl_graph_compiler.hpp"
+#include "dnnl_graph_compiler_version.h"
```

Comment:
Is it required to have a separate header just for the version? Could the version be declared in the same API header - https://github.com/intel-ai/oneDNN/blob/main/src/graph/backend/elyzor/include/dnnl_graph_compiler.h ?

Reply:
We use a [configuration file](https://github.com/intel-ai/oneDNN/pull/2/files#diff-8abe1b7cd1f0d205bd513ebdc8f9391e1c096236ef0929fcf2442821c49b57d3) to generate the version header. I think we can technically include it in the main header as well.